server/routes/info.ts

```diff
@@ -0,0 +1,7 @@
+import { Router } from 'express'
+
+export default function infoRoutes(router: Router) {
+  router.get('pages/info', (req, res) => {
```

Comment:
This one is the URL path, should it be something like `/tiering-information` ?  I'd have suggested `/info` but that's taken - https://tier-dev.hmpps.service.justice.gov.uk/info

Also I guess you're planning to add the `pages/info.njk` file later?

Reply:
Ah okay didn't know that page already existed.

Yeah I'm planning to add the info.njk file later, just wanted to sort out the routes first

Reply:
Changed the reference to `/tiering-additional-information`. Also I did have it without the `pages/` path but still ran into a 404